app/views/marriages/show.html.haml

```diff
@@ -1,5 +1,5 @@
 - content_for(:page_assets) do
-  = ts_tag('marriage')
+  = ts_tag('ujs')
```

Comment:
This pack didn't actually have anything marriage-specific in it. It just imports and initializes Rails UJS. So, I renamed it to `ujs`, so that it can be included elsewhere (as done in this PR) without confusion.